hackerank/js-practice/camelCase.js

```diff
@@ -0,0 +1,11 @@
+function camelcase(s) {
+   let count = 1;
+   for(let i in s){
+    if(s.charAt(i) === s.charAt(i).toUpperCase()){
+      count++
+    }
+   }
+   return count
+}
+
+console.log(camelcase("saveChangesInTheEditor"));
```

Comment:
It's a good practice to add a new line empty at the of file. 

Thas why GH shows the ⛔️ icon in this line.